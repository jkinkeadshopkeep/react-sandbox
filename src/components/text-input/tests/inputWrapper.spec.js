import React from 'react';
import { render } from '@testing-library/react';
import InputWrapper from '../inputWrapper';

describe('TextInputWrapper component', () => {
  let testContext;

  beforeEach(() => {
    testContext = {};
    testContext.component = render(
      <InputWrapper id="foo" label="Dummy Label">
        <span>Dummy child</span>
      </InputWrapper>
    );
  });

  it('renders its children, and a label for an element with the given id', () => {
    expect(testContext.component.asFragment()).toMatchSnapshot();
  });
});
