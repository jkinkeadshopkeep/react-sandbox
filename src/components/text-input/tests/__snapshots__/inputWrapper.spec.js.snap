// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWrapper component renders its children, and a label for an element with the given id 1`] = `
<DocumentFragment>
  <label
    for="foo"
  >
    Dummy Label
  </label>
  <span>
    Dummy child
  </span>
  <br />
</DocumentFragment>
`;
